src/store/api/index.js

```diff
@@ -14,7 +15,9 @@ const state = {
   cloudUrl: process.env.VUE_APP_CLOUD_URL,
   retries: 0,
   serverUrl:
-    localStorage.getItem('server_url') || process.env.VUE_APP_SERVER_URL,
+    settings?.server_url ||
```

Comment:
I wonder about the ordering here? If I change the serverUrl in the home page, it updates but then when I refresh the page it goes back to the one set in settings. Is there a reason why it checks settings before local storage?

Reply:
Hm nope, you're right, that's definitely the incorrect oder. Fixed!